Tools/environment_install/install-prereqs-ubuntu.sh

```diff
@@ -76,8 +77,14 @@ elif [ ${RELEASE_CODENAME} == 'trusty' ]; then
     SITLFML_VERSION="2"
     SITLCFML_VERSION="2"
 else
-    SITLFML_VERSION="2.4"
-    SITLCFML_VERSION="2.4"
+    # Extract the floating point number that is the version of the libcsfml package.
+    SITLCFML_VERSION=`dpkg-query --search libcsfml-audio | cut -d":" -f1 | grep libcsfml-audio | head -1 | grep -Eo '[+-]?[0-9]+([.][0-9]+)?'`
+    # And same for libsfml-audio.
+    SITLFML_VERSION=`dpkg-query --search libsfml-audio | cut -d":" -f1 | grep libsfml-audio | head -1 | grep -Eo '[+-]?[0-9]+([.][0-9]+)?'`
+    # We are on non-Ubuntu so simulate effect of installing pkg-config-arm-linux-gnueabihf.
+    sudo ln -s /usr/share/pkg-config-crosswrapper /usr/bin/arm-linux-gnueabihf-pkg-config
```

Comment:
maybe it would be better to check for pkg-config-arm-linux-gnueabihf first ?

Reply:
That's a good idea, I'll see if I can push a commit amend to this... I'm not really sure how Github works so it may take me some time!

Reply:
you will just need to force push by appening -f to your git push !

Reply:
Thanks. It worked. :-)

I rewrote the pkg-config stuff so it made more sense.

Reply:
Was a commit amend the right thing to do here? Apologies if I got it wrong.